package.json

```
@@ -11,19 +11,22 @@
     "dev:server": "cd pyflask && python -m flask run --port 4242",
     "build": "npm run build:app",
     "build:app": "electron-vite build",
+    "build:mac": "npm run build && npm run build:python-unix && npm run build-mac",
+    "build:win": "npm run build && npm run build:python && npm run build-win",
+    "build:python": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json;. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
+    "build:python-unix": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json:. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
+    "build-win": "electron-builder build --win --publish never",
+    "build-mac": "electron-builder build --mac --publish never",
+    "build-linux": "electron-builder build --linux --publish never",
```

```suggestion
    "build:mac": "npm run build && npm run build:python-unix && npm run build-mac",
    "build:win": "npm run build && npm run build:python && npm run build-win",
    "build:python": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json;. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
    "build:python-unix": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json:. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
    "build:electron:win": "electron-builder build --win --publish never",
    "build:electron:mac": "electron-builder build --mac --publish never",
    "build:electron:linux": "electron-builder build --linux --publish never",
```

suggestion for rename on the final build step (electron; merging the electron-vite plus the flaskdist together)